Signal/src/call/PeerConnectionClient.swift

```diff
@@ -444,13 +444,13 @@ class PeerConnectionClient: NSObject, RTCPeerConnectionDelegate, RTCDataChannelD
         }
     }
 
-    public func addIceCandidate(_ candidate: RTCIceCandidate) {
+    public func addRemoteIceCandidate(_ candidate: RTCIceCandidate) {
         PeerConnectionClient.signalingQueue.async {
             guard self.peerConnection != nil else {
                 Logger.debug("\(self.TAG) \(#function) Ignoring obsolete event in terminated client")
                 return
             }
-            Logger.debug("\(self.TAG) adding candidate")
```

Comment:
Because **so** much is happening in CallService, I thought it would be useful to put all the ICE connectivity logging **just** in PeerConnectionClient. Filtering by "[PeerConnectionClient]" then gives you a succinct view of the ICE handshake without lots of noise.